chainer/functions/theano/theano_function.py

```diff
@@ -0,0 +1,149 @@
+import collections
+
+import numpy
+import six
+
+try:
+    import theano
+    import theano.sandbox.cuda as theano_cuda
+    _available = True
```

Comment:
I thought there is a similar situation to #1309, in which we exposed API that checks availability of `hdf5` or to the check of availability of cuda by `cuda.available`. But when I consider again, I come to think that the situation is different from these two and that your pointing is right. As we do not have expose API until necessary, I now think it is better not to expose this attribute.